website/backend/settings.py

```diff
@@ -29,7 +29,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config('DEBUG', False)
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = config('DJANGO_ALLOWED_HOSTS').split(' ')
```

Comment:
@123MwanjeMike you might want to split by commas. Might be more friendly with .envs